routing/pathfind_test.go

```diff
@@ -630,55 +630,63 @@ func TestNewRoute(t *testing.T) {
 
 	createHop := func(baseFee lnwire.MilliSatoshi,
 		feeRate lnwire.MilliSatoshi, 
-		capacity btcutil.Amount) (*ChannelHop) {
+		capacity btcutil.Amount,
+		timeLockDelta uint16) (*ChannelHop) {
 
 		return &ChannelHop {
 			ChannelEdgePolicy: &channeldb.ChannelEdgePolicy {
 				Node: &channeldb.LightningNode{},
 				FeeProportionalMillionths: feeRate,
 				FeeBaseMSat: baseFee,
+				TimeLockDelta: timeLockDelta,
 			},
 			Capacity: capacity,
 		}
 	}
 
 	testCases := []struct { 
-		name 		    string
-		hops 	    	    []*ChannelHop
-		paymentAmount 	    lnwire.MilliSatoshi
-		expectedFees	    []lnwire.MilliSatoshi
-		expectedTotalAmount lnwire.MilliSatoshi
-		expectError	    bool
-		expectedErrorCode   errorCode
+		name 		      string
+		hops 	    	      []*ChannelHop
+		paymentAmount 	      lnwire.MilliSatoshi
+		expectedFees	      []lnwire.MilliSatoshi
+		expectedTimeLocks     []uint32
```

Comment:
What exactly is the expected time lock here? I'm having a hard time following this usage, but everything else is pristine. It may help to add a set of comments to all the fields in this struct.

Reply:
Comments added